src/app/services/keycloak.service.ts

```diff
@@ -153,9 +153,27 @@ export class KeycloakService {
    * @memberof KeycloakService
    */
   isAllowed(service): boolean {
-    if (service !== 'lock-records') {
+    // admin only routes
+    let adminOnlyRoutes = [
+      'lock-records',
+      'review-data',
+    ]
+    if (!adminOnlyRoutes.find(route => route === service)) {
       return true;
     }
+
+    // block these routes in prod
+    let blockedInProd = [
+      'review-data'
+    ]
+
+    if (this.configService['configuration'].ENVIRONMENT === 'prod') {
+      // we are in prod
+      if (blockedInProd.find(route => route === service)) {
+        return false;
+      }
+    }
+
```

Comment:
We should probably set a specific env var like `this.configService['configuration'].IS_FEATURE_X_ENABLED === 'prod'` vs. checking against the specific environment. Won't require a new build as a result.

Reply:
Removed this env checker in `4c7dc4d`. Can toggle features with a feature flag in the future if necessary.